dlrs/main/lwc/rollupEditor/rollupEditor.html

```diff
@@ -1,269 +1,542 @@
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
+                      type="number"
+                      name="rollup_RowLimit__c"
+                      data-name="rollup_RowLimit__c"
+                      label="Row Limit"
+                      value={rollup.RowLimit__c}
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.RowLimit__c}
+                    ></c-rollup-editor-error>
+                  </div>
+                </div>
+
+                <div class="slds-col slds-size_1-of-1 slds-medium-size_1-of-2">
+                    <!-- Active Toggle -->
+                    <div class="slds-m-top_x-small">
+                        <lightning-input
+                          type="toggle"
+                          name="rollup_Active__c"
+                          data-name="rollup_Active__c"
+                          label="Rollup Status"
+                          checked={rollup.Active__c}
+                          variant="label-stacked"
+                          style="padding-left: 0px !important"
+                        ></lightning-input>
+                        <c-rollup-editor-error
+                          errors={errors.Active__c}
+                        ></c-rollup-editor-error>
+                    </div>
+
+                  <!-- Calculation mode -->
+                  <div class="slds-m-top_x-small">
+                      <lightning-combobox
+                        name="rollup_CalculationMode__c"
+                        data-name="rollup_CalculationMode__c"
+                        label="Calculation Mode"
+                        placeholder="Select"
+                        options={calculationModes}
+                        value={rollup.CalculationMode__c}
+                        required
+                      ></lightning-combobox>
+                      <c-rollup-editor-error
+                        errors={errors.FieldToOrderBy__c}
+                      ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Calculation Sharing Mode -->
+                  <div class="slds-m-top_x-small">
+                      <lightning-combobox
+                        name="rollup_CalculationSharingMode__c"
+                        data-name="rollup_CalculationSharingMode__c"
+                        label="Calculation Sharing Mode"
+                        placeholder="Select"
+                        options={calculationSharingModes}
+                        value={rollup.CalculationSharingMode__c}
+                        required
+                      ></lightning-combobox>
+                      <c-rollup-editor-error
+                        errors={errors.FieldToOrderBy__c}
+                      ></c-rollup-editor-error>
+                  </div>
+
+                </div>
+
+            </div>
+          </div>
+        </div>
+
+        <!-- Text Lookups Section -->
```

Comment:
This feels odd to be a section by itself with a single field. I realize Rollup Details is already a congested section but it might be most appropriate to add it there.

Reply:
I just kept the layout and sections the same as the old page. It should be placed in Rollup Details section as you suggest